recipe/compile_examples.sh

```diff
@@ -31,4 +30,12 @@ cp -vf "$SRC_DIR/lib/libyaml-cpp.a" "$PREFIX/lib/"
 
 cmake -S $PREFIX/examples ${CMAKE_ARGS}
 
-cmake --build .
+if [[ ${target_platform} == "linux-64" ]]; then
+  cmake --build . -j"$(nproc)"
+else
+  cmake --build . --target hello_world -j"$(nproc)"
+  cmake --build . --target video_replayer -j"$(nproc)"
+  cmake --build . --target activation_map -j"$(nproc)"
+  cmake --build . --target bring_your_own_model -j"$(nproc)"
+  cmake --build . --target holoviz -j"$(nproc)"
```

Comment:
Can we also add the MatX example (new in 3.6)?

```suggestion
    cmake --build . --target matx_basic -j"$(nproc)"
```